spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+    Test.setup_models
+    Rails.application.load_seed
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
```

Comment:
do we want to use this strategy for all capybara specs? I did some research on db cleaner and then got in a little over my head http://stackoverflow.com/questions/11419536/postgresql-truncation-speed/11423886#11423886